docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md

```diff
@@ -18,6 +18,12 @@ Clients in Camunda 8 SaaS and applications in Camunda 8 Self-Managed provide a s
 
 ## Steps
 
+:::caution Keycloak URLs
+As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect customers who have blocked the Keycloak frontend URL from other services, including Camunda applications, and can potentially impact Identity's functionality.
```

Comment:
```suggestion
As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect you if you have blocked the Keycloak frontend URL from other services (including Camunda applications), and can potentially impact Identity's functionality.
```
Suggestion to address the customer directly in second person, rather than in the third person.